app/templates/collection/view.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="container dwcm-collectionobject-view">
+    <div class="page-header">
+        <h1>
+            {{model.collectionName}}{{if model.catalogNumber (concat ' / ' model.catalogNumber) ''}}
```

Comment:
Looks like some leftovers from the collection object view? Don't think a collection actually has a catalogNumber.